companion/src/firmwares/modeldata.h

```diff
@@ -121,6 +121,7 @@ class ModelData {
     int       trimInc;            // Trim Increments
     unsigned int trimsDisplay;
     bool      disableThrottleWarning;
+    unsigned int jitterFilter;
```

Comment:
Please add comment something like "Introduced in Edge 2.7 Yaml only". This helps explain why not in opentxeeprom.cpp import/export especially since the old global setting existed from v218.

Reply:
OK